src/Chainweb/BlockHeader/Genesis.hs

```diff
@@ -139,43 +148,51 @@ genesisBlockTarget _ _ = maxTarget
 --
 -- It holds that:
 --
--- prop> mainnet20InitialHashTarget == HashTarget . (4 *) <$> (runGet decodePowHashNat "DOordl9cgfs4ZTBdFnbjRW5th-hW-pL33DIAAAAAAAA")
+-- prop> Just mainnet20InitialHashTarget == HashTarget . (4 *) <$> (runGet decodePowHashNat =<< decodeB64UrlNoPaddingText "DOordl9cgfs4ZTBdFnbjRW5th-hW-pL33DIAAAAAAAA")
 --
 mainnet20InitialHashTarget :: HashTarget
-mainnet20InitialHashTarget = HashTarget 92812039490652836170182663013446053204096613861175006070293989356886611016976
+mainnet20InitialHashTarget = HashTarget 0x000000000000cb73de4be95ba21db5b9178dd85974c194e3ee05717dd8afa830
```

Comment:
I verified that this correctly converts to what we had in the previous commit.

Reply:
Also verified that mainnet currently has a target with 12 leading zeroes, so this is the right ball park.